README.md

```diff
@@ -109,6 +123,36 @@ You can run the application with this command: `air`.
 
 It will generate and reload thyra each time a file is modified.
 
+### Plugin installation
+
+Thanks to its modular architecture, it is possible to use third party plugins in Thyra.
+Plugins must be placed in a `plugins` folder in the Thyra config directory: `$HOME/.config/thyra`.
+Some plugins are provided by Massa and the following procedure describe the Playground plugin as an example.
+
+#### Playground plugin installation
+
+Download the plugin:
+
+-   [MacOS](https://github.com/massalabs/thyra-playground-plugin/suites/9125906945/artifacts/423809352)
+-   [Linux](https://github.com/massalabs/thyra-playground-plugin/suites/9125906945/artifacts/423809353)
+-   [Windows](https://drive.google.com/file/d/1vAXZC4eGVJ-TALWj_eyAHSnTsEmiICcw/view?usp=share_link)
+
+create the `plugins` directory and unzip the content of the archive in it.
+
+Your plugins directory should look like this:
+```
+  plugins
+    └── playground-plugin
+        ├── manifest.json
+        ├── simulator
+        │   └── massa-sc-tester
```

Comment:
why do we need the simulator?

Reply:
because the idea of the plugin is tu be able to use simulator from the playground.
But dont worry simulator is already embeded in the playground-plugin. so no need to take care of it